src/MongoDB.EntityFrameworkCore/Metadata/MongoAnnotationNames.cs

```diff
@@ -55,4 +55,44 @@ public static class MongoAnnotationNames
     /// The key for create index options annotations.
     /// </summary>
     public const string CreateIndexOptions = Prefix + nameof(CreateIndexOptions);
+
+    /// <summary>
+    /// The key for the id of the data key used for encryption.
+    /// </summary>
+    public const string EncryptionDataKeyId = Prefix + nameof(EncryptionDataKeyId);
```

Comment:
This one isn't prefixed with Queryable as we might add support for CSFLE which could re-use it.